apps/autostart-stop.html.markerb

```diff
@@ -46,29 +56,25 @@ Default settings in `fly.toml` for V2 Apps created using the `fly launch` comman
 ...
 ```
 
-<div class="callout">
-Existing V2 Apps&mdash;or any V2 Apps that don't have these settings in `fly.toml`&mdash;have the default values `auto_start_machines = true` and `auto_stop_machines = false`.
-</div>
-
-In general, we recommend setting `auto_stop_machines` and `auto_start_machines` to the same value to avoid having Machines that either never start or never stop. 
+### Recommended settings
 
-If `auto_start_machines = true` and `auto_stop_machines = false`, then Fly Proxy will automatically start your Machines but will never stop them. This means you'll have to stop Machines manually, or have your [app exit when idle](#stop-a-machine-by-terminating-its-main-process).
+If your app already [exits when idle](#stop-a-machine-by-terminating-its-main-process), then you can set `auto_start_machines = true` and `auto_stop_machines = false` to have Fly Proxy automatically restart the Machines stopped by your app. 
 
-If `auto_start_machines = false` and `auto_stop_machines = true`, then Fly Proxy will automatically stop your Machines when there's low traffic, but won't be able to start them again. If all or most of your Machines are stopped, then requests to your app will start failing.
+If your app doesn't exit when idle, then we recommend setting `auto_stop_machines` and `auto_start_machines` to the same value (either both `true` or both `false`) so that Fly Proxy doesn't stop Machines and never restart them. For example, if `auto_start_machines = false` and `auto_stop_machines = true`, then Fly Proxy automatically stops your Machines when there's low traffic but doesn't start them again. When all or most of your Machines are stopped, requests to your app will start failing.
 
-You can set `min_machines_running` to `1` or higher if you need at least one instance of your app running all the time. This setting is for the total number of Machines, not Machines per region. For example, if `min_machines_running = 1`, then your app will scale down until there is only one Machine running in your primary region. 
+When `auto_stop_machines = true`, set `min_machines_running` to `1` or higher if you need to keep one or more Machines running all the time in your primary region. The `min_machines_running` value applies to the total number of Machines, not Machines per region. For example, if `min_machines_running = 1` and there's no traffic to your app, then Fly Proxy will stop Machines until eventually there is only one Machine running in your primary region.
 
 There's no "maximum machines running" setting, because the maximum number of Machines is just the total number of Machines you've created for your app. Learn more in the [How It Works](#how-it-works) section.
```

Comment:
Nitpick: "How it works" capitalization changed, perhaps making it consistent here would make sense. 

[How it works](#how-it-works) section.